apps/examples/st_things/tests/sample_tests_things.c

```diff
@@ -0,0 +1,184 @@
+	set_rep_push_mesg();
+}
+#endif
+
+static void handle_reset_result(bool result)
+{
+	printf("[%s]Reset %s.\n", result ? "succeeded" : "failed", TAG);
+}
+
+static void handle_things_status_change(st_things_status_e things_status)
+{
+	printf("[%s]Things status is changed: %d\n", TAG, things_status);
+}
+
+static bool handle_reset_request(void)
+{
+	printf("[%s]Received a request for RESET.\n", TAG);
+	return true;
+}
+
+static bool handle_ownership_transfer_request(void)
+{
+	printf("[%s]Received a request for Ownership-transfer. \n", TAG);
+	return true;
+}
+
+static bool handle_get_request(st_things_get_request_message_s *req_msg, st_things_representation_s *resp_rep)
+{
+	printf("Received a GET request on %s\n", req_msg->resource_uri);
+
+	if (0 == strncmp(req_msg->resource_uri, g_res_switch, strlen(g_res_switch))) {
+		return handle_get_request_on_switch(req_msg, resp_rep);
+	} else if (0 == strncmp(req_msg->resource_uri, g_res_dimming, strlen(g_res_dimming))) {
+		return handle_get_request_on_dimming(req_msg, resp_rep);
+	} else if (0 == strncmp(req_msg->resource_uri, g_res_temp, strlen(g_res_temp))) {
+		return handle_get_request_on_ct(req_msg, resp_rep);
+	} else {
+		printf("Not supported uri.\n");
+	}
+	return false;
+}
+
+static bool handle_set_request(st_things_set_request_message_s *req_msg, st_things_representation_s *resp_rep)
+{
+	printf("Received a SET request on %s\n", req_msg->resource_uri);
+
+	if (0 == strncmp(req_msg->resource_uri, g_res_switch, strlen(g_res_switch))) {
+		return handle_set_request_on_switch(req_msg, resp_rep);
+	} else if (0 == strncmp(req_msg->resource_uri, g_res_dimming, strlen(g_res_dimming))) {
+		return handle_set_request_on_dimming(req_msg, resp_rep);
+	} else if (0 == strncmp(req_msg->resource_uri, g_res_temp, strlen(g_res_temp))) {
+		return handle_set_request_on_ct(req_msg, resp_rep);
+	}
+	return false;
+}
+
+int ess_process(void)
+{
+#ifdef CONFIG_RESET_BUTTON
+	if (!check_reset_button_pin_number()) {
+		printf("Error : Invalid pin number.\n");
+		return 0;
+	}
+	iotapi_initialize();
+	iotbus_gpio_context_h m_gpio = iotbus_gpio_open(CONFIG_RESET_BUTTON_PIN_NUMBER);
+	iotbus_gpio_register_cb(m_gpio, IOTBUS_GPIO_EDGE_RISING, gpio_callback_event, (void *)m_gpio);
+#endif
+
+#ifdef CONFIG_LIGHT_SWITCH_BUTTON
+	iotbus_gpio_context_h e_gpio = iotbus_gpio_open(CONFIG_LIGHT_SWITCH_BUTTON_PIN_NUMBER);
+	iotbus_gpio_register_cb(e_gpio, IOTBUS_GPIO_EDGE_RISING, gpio_callback_event_trigger_light_switch, (void *)e_gpio);
+#endif
+
+	bool easysetup_complete = false;
+
+        
+        if (st_things_start() == ST_THINGS_ERROR_NONE) {
+                printf("[%s]=====================================================\n", TAG);
+                printf("[%s]                    ST_Things_SDK_TC_2 FAIL                    \n", TAG);
+                printf("[%s]=====================================================\n", TAG);
+        }
+        else
+        {
```

Comment:
Opening brace should go on the same line of 'if', else', 'while', 'switch' and all conditional blocks.